xGame/client/src/Components/Match/Match.js

```diff
@@ -36,7 +36,7 @@ const Match = (props) => {
         if (piece) {
           const newPiece = {
             pieceName: getPieceName(piece.color, piece.type),
-            location: piece.position
+            location: piece.position,
```

Comment:
Can you remove the trailing commas in this file? It's probably an autosave setting that needs to be turned off in IntelliJ